verification/store_test.go

```diff
@@ -16,36 +19,61 @@ func TestLoadValidTrustStore(t *testing.T) {
 }
 
 func TestLoadSymlinkTrustStore(t *testing.T) {
-	_, err := LoadX509TrustStore("testdata/trust-store/valid-trust-store_SYMLINK")
-	if err == nil || err.Error() != "\"testdata/trust-store/valid-trust-store_SYMLINK\" is not a regular directory (symlinks are not supported)" {
+	// TODO run symlink tests on Windows
+	if runtime.GOOS == "windows" {
+		t.Skip("skipping the symlink test on Windows")
+	}
+	path := filepath.FromSlash("testdata/trust-store/valid-trust-store_SYMLINK")
+	_, err := LoadX509TrustStore(path)
+	fmt.Println(err)
+	if err == nil || err.Error() != fmt.Sprintf("%q is not a regular directory (symlinks are not supported)", path) {
 		t.Fatalf("symlinks should return error : %q", err)
```

Comment:
This is expected. Since the if condition is checking whether the err is nil or the error message is not matching, it is good to know to which is the case by printing the err.

Reply:
We can print both the expected error and the returned error. Like what we do at https://github.com/notaryproject/notation-go/blob/main/plugin/metadata_test.go#L25.

Reply:
Wouldn't that still print `%!q(<nil>)` as the returned err is still a nil? The link you sent is printing booleans, not errors.